### DISCUSSION TITLE
Maximum Chunks & File Size

### DISCUSSION THREAD

**Discussion by mousedown** [vercel/next.js#12983] [Help]
I've been optimising my site (First Contentful Paint mainly). I started going through the code base, having a big clean and tidy. Lightening the DOM , removing unused CSS, deleting redundant components etc. Basically, just removing things. Lots of things. Nothing was introduced,  I was amazed to see that the next output stats were made worse. More chunks were created, and the First Load sizes for all increased. Experimental ,  `granularChunks: true` did nothing. 

 Is this expected behaviour?  Can anyone shed any light on how this could be possible. I get that there must be some duplication across the chunks which I'm trying to avoid. 

![Screenshot 2020-05-16 at 12 39 27](https://user-images.githubusercontent.com/491789/82126273-9f855400-97a3-11ea-8835-7f6796d10f5f.png)
![Screenshot 2020-05-16 at 18 16 09](https://user-images.githubusercontent.com/491789/82126279-a44a0800-97a3-11ea-831f-3f20672c5bee.png)

**Comment by Huespal** [comment]
We are in the same boat.

NextJS 9.4.4, react, redux, sagas, styled components, hotjar and GTM. Some custom fonts, no image assets. Very little code in pages.

![Build](https://user-images.githubusercontent.com/4157755/84126566-5e006580-aa3e-11ea-9b93-dcae1d83c4cc.png)

'node_modules' chunk -> ~250Kb

I don't understand how to get < 130Kb to avoid those red numbers.

Any help is appreciated.

P.D: experimental granularChunks did nothing.
